api/v1alpha1/osbuildconfig_types.go

```diff
@@ -63,6 +63,17 @@ type BuildDetails struct {
 	TargetImage TargetImage `json:"targetImage"`
 }
 
+// EdgeInstallerBuildDetails includes all the information needed to build the edge-installer image
+type EdgeInstallerBuildDetails struct {
+	// Distribution is the name of the O/S distribution
+	Distribution string `json:"distribution"`
+	// OSTree is the OSTree configuration of the build (optional)
+	OSTree *OSTreeConfig `json:"osTree,omitempty"`
```

Comment:
I think that for the EdgeInstaller the OSTreeConfig is mandatory

Reply:
I'm not sure about that - I looked into the handler code and I didn't find such a validation 
but I think we can require it from the user

Reply:
First, it's not from the user since this struct is only in the OSBuild which is created by the controller. Second, the reason it is not mandatory in the handler is that they use default values if not provided. But, in this specific case, the idea is to build an edge-installer from a previously built edge-container. So, we must provide these details.